worldengine/cli/main.py

```diff
@@ -381,7 +393,7 @@ def main():
     sys.setrecursionlimit(args.recursion_limit)
 
     if args.number_of_plates < 1 or args.number_of_plates > 100:
-        usage(error="Number of plates should be in [1, 100]")
+        usage(error="Number of plates should be a in [1, 100]")
```

Comment:
I'm not sure what that means... my local repo says I'm up to date with master. I was puzzled by these changes too.